TestAssets/TestProjects/WebAppWithMissingFileInPublishOptions/.gitignore

```diff
@@ -0,0 +1,234 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
I just copied the test asset and modified what I wanted. This is in the template by default.